app/models/message.rb

```diff
@@ -153,21 +154,12 @@ def push_event_data
     merge_sender_attributes(data)
   end
 
-  def search_data
-    data = attributes.symbolize_keys
-    data[:conversation] = conversation.present? ? conversation_push_event_data : nil
-    data[:attachments] = attachments.map(&:push_event_data) if attachments.present?
-    data[:sender] = sender.push_event_data if sender
-    data[:inbox] = inbox
-    data
-  end
-
   def conversation_push_event_data
     {
       assignee_id: conversation.assignee_id,
       unread_count: conversation.unread_incoming_messages.count,
       last_activity_at: conversation.last_activity_at.to_i,
-      contact_inbox: { source_id: conversation.contact_inbox.source_id }
+      contact_inbox: conversation.contact_inbox ? { source_id: conversation.contact_inbox.source_id } : nil
```

Comment:
Why was this change added? At the moment, there shouldn't be a case where contact_inbox is nil.